Container/HTML/textUtilities.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Container/CSS/textUtilities.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- Heading -->
+    <h1>H1 - Heading</h1>
+    <h2>H2 - Heading</h2>
+    <h3>H3 - Heading</h3>
+    <h4>H4 - heading</h4>
+    <h5>H5 - Heading</h5>
+
+    <!-- small text -->
+    <h3>Small Text</h3>
+    <h3 class="small-text">
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Officiis
+      delectus, consequatur nesciunt numquam rem illum? A assumenda dolor
+      explicabo nulla voluptates dicta natus non repellendus modi recusandae ex,
+      mollitia nemo!
+    </h3>
```

Comment:
small text's size is the same as regular text. you haven't defined CSS for small-text